default_app/renderer.js

```diff
@@ -4,49 +4,60 @@ const path = require('path')
 const URL = require('url')
 const electronPath = path.relative(process.cwd(), remote.process.execPath)
 
-Array.from(document.querySelectorAll('a[href]')).forEach(link => {
-  // safely add `?utm_source=default_app
-  let url = URL.parse(link.getAttribute('href'), true)
-  url.query = Object.assign(url.query, {utm_source: 'default_app'})
-  url = URL.format(url)
-
-  link.addEventListener('click', (e) => {
-    e.preventDefault()
-    shell.openExternal(url)
+function initialize () {
+  Array.from(document.querySelectorAll('a[href]')).forEach(link => {
```

Comment:
While you're here, can you please change it to
```js
for (const link of document.querySelectorAll('a[href]'))) {
 ...
}
```

All those `Array.from` and `.forEach` only create noise =/

Reply:
It's not related to the PR topic, so it would be better in a cleanup PR.

Agreed on the sentiment tho

Reply:
Nobody ever makes `cleanup PR` =/